learningresources/models.py

```diff
@@ -27,6 +28,12 @@
 STATIC_ASSET_PREFIX = 'assets'
 STATIC_ASSET_BASEPATH = STATIC_ASSET_PREFIX + '/{org}/{course_number}/{run}/'
 
+# Used to signify that a repository has been created
+# and its initial permissions have been set.
+# This allows the "curation status" vocabulary to be
+# created automatically.
+repo_created = Signal(providing_args=["repository"])
```

Comment:
why this has been defined here and not in the signals? (asking because I actually don't know about best practices in this case)